kyo-zio/shared/src/main/scala/kyo/zios.scala

```diff
@@ -0,0 +1,71 @@
+package kyo
+
+import core.*
+import core.internal.*
+import kyo.fibersInternal.*
+import scala.annotation.targetName
+import scala.util.control.NonFatal
+import zio.IO
+import zio.Task
+import zio.UIO
+import zio.URIO
+import zio.ZEnvironment
+import zio.ZIO
+
+class ZIOs extends Effect[ZIOs]:
+    type Command[T] = Task[T]
+
+object ZIOs extends ZIOs:
+
+    def run[T: Flat](v: T < (Fibers & ZIOs)): Task[T] =
+        def loop(v: T < (Fibers & ZIOs)): Task[T] =
+            v match
+                case kyo: Suspend[?, ?, ?, ?] =>
```

Comment:
There's no need to handle other effects here. This method handles these effects as the "last ones". `Aborts` for example needs to be handled before calling it. We need this multi handler because `IOs`, `Fibers`, and `Zios` require deep handlers, which is essentially a regular [monad interface](https://github.com/getkyo/kyo/blob/d89f10fd0b7ab6182d43d038918648da844749b7/kyo-core/shared/src/main/scala/kyo/core.scala#L181).

Reply:
Right... my suggestion was to enable handling `Aborts` via a mapping to `ZIO.fail`, though I think the problem may be with multiple aborts.

Reply:
I see! It makes sense to add `Aborts[Throwable]` since `Task` supports failures. I've added it by handling `Aborts` first and using ZIO's `flatMap`.